tests/foreman/ui/test_contenthost.py

```diff
@@ -135,11 +172,27 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    property_name = setting_update.name
+    _distro = module_repos_collection_with_manifest.distro
+    host_rhel_version = None
+    if _distro.startswith('rhel'):
+        host_rhel_version = int(_distro[4:])
+    rhel_status = get_rhel_lifecycle_support(host_rhel_version if not None else 0)
+
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     startdate = datetime.utcnow().strftime('%m/%d/%Y')
     with session:
+        session.settings.update(f'name = {property_name}', 'Yes')
         session.location.select(default_location.name)
+        session.organization.select(module_org.name)
+        # Use new hosts UI, only check the details and host status
+        host_details = session.host.get_details(vm.hostname, widget_names='properties')
+        host_status = session.host_new.get_host_statuses(vm.hostname)
+        # Use old hosts UI for remainder
```

Comment:
Why do you need to use both new UI and old UI? Is it because you can't do the whole workflow in the new one? If that's the case, I think it's a major UX bug.

If it's because lack of support in Airgun, I think it would be better to either use only old UI, or (preferably) add the support to Airgun.

Reply:
Can't do the whole workflow in new UI yet, as the contenthost parts fails navigation with new UI, I believe there will need to be some airgun fixes, not completely sure why it won't pass with new UI setting enabled through the entire test.
This test is a bit older written for old UI, so I think for an extension we should keep this on the old UI?
We could plan to update this to use nav in `host_new.py` in the future